frontend/admin/src/pages/home/PasswordChange.tsx

```diff
@@ -37,12 +39,16 @@ const PasswordChange = () => {
             oldPassword,
             newPassword
         })
-            .then(() => history.push("/"))
+            .then(() => {
+                history.push("/")
+                toast('Passwort ändern', 'Passwort wurde geändert.');
```

Comment:
I'd switch the order. `history.push()` suggests that you leave this page so you should not perform any more actions after leaving.